.phpunit.result.cache

```diff
@@ -1 +1 @@
-{"version":1,"defects":{"BasicTestCase::testCase":3,"App\\Tests\\BasicTestCase::testCase":3,"App\\Tests\\UserTest::testAdminCreateUser":6,"App\\Tests\\UserTest::testRegisterUser":6,"App\\Tests\\UserTest::testUserConfirmation":4,"App\\Tests\\UserTest::testMinimumRole":3,"App\\Tests\\UserTest::testSendConfirmation":3,"App\\Tests\\UserCheckerTest::testCheckPreAuthFailsIsEnabled":3,"App\\Tests\\UserCheckerTest::testCheckPreAuthFailsIsConfirmed":3},"times":{"App\\Tests\\PagesAreFoundTest::testPublicPagesAreFound with data set #0":2.637,"App\\Tests\\PagesAreFoundTest::testLoggedInPagesAreFound with data set #0":0.218,"BasicTestCase::testCase":0.002,"App\\Tests\\BasicTestCase::testCase":0.002,"App\\Tests\\UserTest::testAdminCreateUser":0.001,"App\\Tests\\UserTest::testRegisterUser":0.006,"App\\Tests\\UserTest::testUserConfirmation":0,"App\\Tests\\UserTest::testMinimumRole":0.002,"App\\Tests\\UserTest::testUserRequestPassword":0,"App\\Tests\\UserTest::testEachUserHasAtLeastOneRole":0,"App\\Tests\\UserTest::testRolesAreCleanForDisplay":0,"App\\Tests\\UserTest::testSetNewPassword":0,"App\\Tests\\UserTest::testConfirmUser":0,"App\\Tests\\UserTest::testSendConfirmation":0.005,"App\\Tests\\UserCheckerTest::testCheckPreAuthFailsIsEnabled":0.002,"App\\Tests\\UserCheckerTest::testCheckPreAuthFailsIsConfirmed":0.009,"App\\Tests\\UserCheckerTest::testCheckPreAuthFailsIfUnconfirmed":0.011,"App\\Tests\\UserCheckerTest::testCheckPreAuthFailsIfEnabled":0.002,"App\\Tests\\UserCheckerTest::testCheckPostAuthErasesPasswordRequest":0}}
```

Comment:
Moet deze file niet in .gitignore? En dan ook verwijderen uit git?

Reply:
done, [0c437e1](https://github.com/sumocoders/Framework-User-Implementation-Example/pull/11/commits/0c437e1f3e1284ed37f8ef138d82d861a583ccc0)